jdaviz/core/template_mixin.py

```diff
@@ -1823,6 +1833,9 @@ def is_trace(lyr):
         def not_trace(lyr):
             return not is_trace(lyr)
 
+        def is_dq_layer(lyr):
+            return 'DQ' in getattr(getattr(lyr, 'data', None), 'label', '')
```

Comment:
@bmorris3 - is there a better general way to detect this, perhaps based on something in the metadata or units?  Without a filter on this, children that aren't DQ layers are causing the DQ plugin to hang.

Reply:
We could add a DQ metadata entry label at parse time, which would identify the data with something other than its label (also given at the parser). Right now, there's no guaranteed-to-work alternative.